docs/next.config.js

```diff
@@ -171,6 +171,7 @@ module.exports = {
     PULL_REQUEST: process.env.PULL_REQUEST === 'true',
     REACT_STRICT_MODE: reactStrictMode,
     FEEDBACK_URL: process.env.FEEDBACK_URL,
+    SITE_NAME: process.env.SITE_NAME,
```

Comment:
It already exists, it's `NETLIFY_SITE_NAME`.

```suggestion
```

Reply:
Removed in favor of `STAGING`.